.editorconfig

```diff
@@ -1,12 +1,8 @@
+# Editorconfig file for cross-platform configuration of editors.
 root=true
 
-[*.ini]
-indent_style=space
-indent_size=4
-
-[*.{yml,yaml}]
-indent_style=space
-indent_size=2
+[*.py]
+max_line_length=79
```

Comment:
I'd vote for 119 line length. @msb ? @rjw57 ?

Reply:
We would have to add configuration to flake8 to ignore line length since it enforces PEP8: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
And I don't know off the top of my head if flake8 supports a configurable line length as opposed to just ignoring line length errors.

Reply:
And even if we make use of PEP8's wiggle room, to paraphrase Monty Python and the Holy Grail "99 is the maximum line length and the length of the maximum line shall be 99. 112 is right out!"

> Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters (effectively increasing the maximum length to 99 characters), provided that comments and docstrings are still wrapped at 72 characters.

Reply:
The consensus appears to be "99".